src/d3d11/d3d11_view_srv.cpp

```diff
@@ -144,13 +144,14 @@ namespace dxvk {
           viewInfo.numLayers = 1;
           break;
           
-        case D3D11_SRV_DIMENSION_TEXTURECUBE:
-          viewInfo.type      = VK_IMAGE_VIEW_TYPE_CUBE_ARRAY;
+        case D3D11_SRV_DIMENSION_TEXTURECUBE: {
```

Comment:
Why did you add braces? Seems unnecessary and inconsistent to me.

Reply:
because he defines a variable inside the case.